### ISSUE TITLE
question about data download ?

### ISSUE THREAD

**Issue by quanxiang-liu** [TankZhouFirst/clinical-grade-computational-pathology-using-weakly-supervised-deep-learning-on-whole-slide-images#1]
When I uese download_dataset.py to download your public data，Even I use ss ，still can't success.

Error："HTTP status code=403, Exception=Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature."

On the Tencent Cloud ，I download SVS but can't find "target.csv"，please upload csv.

Thanks

**Comment by TankZhouFirst** [comment]
wait a moment, i`ll check it

**Comment by TankZhouFirst** [comment]
> When I uese download_dataset.py to download your public data，Even I use ss ，still can't success.
> 
> Error："HTTP status code=403, Exception=Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature."
> 
> On the Tencent Cloud ，I download SVS but can't find "target.csv"，please upload csv.
> 
> Thanks

wait a moment, i`ll check it

**Comment by TankZhouFirst** [comment]
> When I uese download_dataset.py to download your public data，Even I use ss ，still can't success.
> 
> Error："HTTP status code=403, Exception=Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature."
> 
> On the Tencent Cloud ，I download SVS but can't find "target.csv"，please upload csv.
> 
> Thanks

i have just uploaded the target.csv file, please check it

**Comment by quanxiang-liu** [comment]
OK，I got it
Thanks




On 08/22/2019 09:00, TankZhou wrote:

When I uese download_dataset.py to download your public data，Even I use ss ，still can't success.

Error："HTTP status code=403, Exception=Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature."

On the Tencent Cloud ，I download SVS but can't find "target.csv"，please upload csv.

Thanks

i have just upload the target.csv file, please check it

**Comment by zuoyuwei** [comment]
I can't download this datasets on the Tencent Cloud. I hope you can give me a help. Thanks

**Comment by TankZhouFirst** [comment]
> I can't download this datasets on the Tencent Cloud. I hope you can give me a help. Thanks

sorry, because my membership is expired, space is limited in 10G, so i deleted it.

**Comment by TankZhouFirst** [comment]
> I can't download this datasets on the Tencent Cloud. I hope you can give me a help. Thanks
i'll update a new reference link in these days, sorry.

**Comment by zuoyuwei** [comment]
> > I can't download this datasets on the Tencent Cloud. I hope you can give me a help. Thanks
> > i'll update a new reference link in these days, sorry.

Thanks

**Comment by TankZhouFirst** [comment]
> > > I can't download this datasets on the Tencent Cloud. I hope you can give me a help. Thanks
> > > i'll update a new reference link in these days, sorry.
> 
> Thanks

i haved updated the dataset download link, see readme file

**Comment by Sadegh-Saberian** [comment]
Hi,

are the cloud storage still working? I am trying to download the dataset but the it keeps getting failed.

**Comment by TankZhouFirst** [comment]
> Hi,
> 
> are the cloud storage still working? I am trying to download the dataset but the it keeps getting failed.

yes, it still works. But i do not know if it is accessible outside china.

**Comment by Sadegh-Saberian** [comment]
unfortunately it is not accessible outside china. is there any way I can work it out?

**Comment by TankZhouFirst** [comment]
> unfortunately it is not accessible outside china. is there any way I can work it out?

maby you can try this : https://wiki.cancerimagingarchive.net/display/Public/Breast+Metastases+to+Axillary+Lymph+Nodes
